docs/integrations/sources/linkedin-pages-migrations.md

```diff
@@ -0,0 +1,6 @@
+# Linkedin Pages Migration Guide
+
+## Upgrading to 2.0.0
+
+- 2 new streams have been added which support time bound incremental sync. They need additional fields `start_date` and `time_granularity_type` to be defined in the config to work.
```

Comment:
But both those fields are optional, and the previous versions didn't have these 2 streams, so those will work even when these fields are not defined. These fields are needed only if the user wants to use either of the 2 newer streams.
I can set defaults though, if needed.

Reply:
the connector needs either a breaking change or defaults.

This would break if a user upgrades to the new version and enables the new stream without updating their config

Reply:
Ah alright, that makes sense, I'll enable defaults for these fields.

Reply:
I'm not able to use the day_delta() macro to set the default value for start_date (I wanted to set a default start date that was a year prior to the current date), I think that it requires a hardcoded value. So should I make start_date a breaking change? And I can set a default for the time_granularity_type field

Reply:
Added a default start_date value of `2023-01-01T00:00:00Z`, however Linkedin Pages supports a lookback of atmost 12 months, so I added logic to default it to 12 months prior the current date if the start date is before that.